resolwe_bio/processes/expression/featureCount.yml

```diff
@@ -0,0 +1,171 @@
+    - name: id_attribute
+      label: ID attribute
+      type: basic:string
+      default: gene_id
+      description: >
+        GTF attribute to be used as feature ID. Several GTF lines with the same feature ID will be considered as parts of the same feature. The feature ID is used to identity the counts in the output table.
+    - name: format_of_annotation
+      label: Format of the annotation file
+      type: basic:string
+      required: false
+      default: GTF
+      description: >
+        Specify the format of the annotation file. GTF file is used by default.
+    - name: count_multi_mapping_reads
+      label: Count multi mapping reads
+      type: basic:boolean
+      required: false
+      default: false
+      description: >
+        If specified, multi-mapping reads/fragments will be counted.
+    - name: min_mqs
+      label: Minimal mapping quality score
+      type: basic:integer
+      required: false
+      default: 0
+      description: >
+        The minimum mapping quality score a read must satisfy in order to be counted. For paired-end reads, at least one end should satisfy this criteria.
+    - name: strand_specific
+      label: Strand specific read counting
+      type: basic:integer
+      required: false
+      default: 0
+      description: >
+        Indicate if strand-specific read counting should be performed.
+      choices:
+        - label: unstranded
+          value: 0
+        - label: stranded
+          value: 1
+        - label: reversely stranded
+          value: 2
+    - name: feature_type
+      label: Feature type
+      type: basic:string
+      default: exon
+      description: >
+        Feature type (3rd column in GFF file) to be used, all features of other type are ignored.
+    - name: cpu_number
+      label: Number of threads/CPUs
+      type: basic:integer
+      required: false
+      default: 1
+      description: >
+        Specify the number of threads/CPUs used for mapping
+    - name: PE_options
+      label: Paired end alignment options
+      group:
+        - name: is_paired_end
+          label: Is paired end
+          type: basic:boolean
+          required: false
+          default: false
+          description: >
+            If specified, only fragments that have both ends successfully aligned will be considered for summarization.
+        - name: require_both_ends_mapped
+          label: Require both ends mapped
+          type: basic:boolean
+          required: false
+          description: >
+            If specified, only fragments that have both ends successfully aligned will be considered for summarization.
+        - name: count_chimeric_fragments
+          label: Count chimeric fragments
+          type: basic:boolean
+          required: false
+          description: >
+           If specified, the chimeric fragments (those fragments that have their two ends aligned to different chromosomes) will NOT be counted.
+  output:
+    - name: feature_counts_output
```

Comment:
Move this and the next two fields after the standard alignment fields. I guess we should have the "standard" fields first in all aligners so that UI looks consistent.